tools/RAiDER/utilFcns.py

```diff
@@ -1222,3 +1223,36 @@ def get_dt(t1,t2):
      18000.0
     '''
     return np.abs((t1 - t2).total_seconds())
+
+
+def calc_buffer(model:str, lat:float, lon:float):
+    """ Calculate the buffer for ray tracing from the latitude and model
+
+    Quick and dirty estimate
+    """
+    from shapely.geometry import Point
+    from shapely.ops import transform
+    if -90 < lat < 90:
+        proj = False
+    else:
+        proj = True
+        lon, lat = transform_coords(4087, 4326, lon, lat)
+
+    # get the top of the model
+    dct_tom = {'HRES': 80301.65, 'HRRR': 80301.65, 'GMAO': 80301.65, 'ERA5': 80301.65}
+    tom     = dct_tom[model]
+    near_range = tom / np.sin(np.deg2rad(30))
```

Comment:
This should be drive by the angle at near range. This will be much overkill for UAVSAr for padding, so perhaps the 30 degrees is good enough for now.